news-2.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+        <head>
+            <meta charset="UTF-8">
+            <title>Virametrics - High Death Mortality Rate in Us</title>
+        </head>
+        <body>
+            <table>
+            <tr>
+            <th><h1>Virametrics</h1></th>
+            <th><img src="image/v.png"></th>
```

Comment:
the folder name here should be `images`
```suggestion
            <th><img src="images/v.png"></th>
```